app/Main.hs

```diff
@@ -2,11 +2,14 @@
 {-# LANGUAGE TemplateHaskell   #-}
 module Main where
 
+import Prelude
```

Comment:
It's not, I just like having explicit Prelude imports and they snuck in. I'll remove them.